lib/routes/zhihu/posts.js

```diff
@@ -38,7 +38,7 @@ module.exports = async (ctx) => {
                 title: String(item.title),
                 description: item.content,
                 link: item.url,
-                pubDate: new Date(item.updated).toUTCString(),
+                pubDate: new Date(item.updated * 1000).toUTCString(),
```

Comment:
```suggestion
                pubDate: parseDate(item.updated * 1000),
```
please use parseDate()
see https://github.com/DIYgod/RSSHub/issues/6795
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei

Reply:
Got it, thanks!